### DISCUSSION TITLE
BT-722-Contract in notice subType 38

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#470] [Q&A] [answered]
Hi lab
I did not check in sdk 1.7, in sdk 1.6 I think there is an error : 

in 38.json : 

 "id" : "BT-722-Contract",
            "contentType" : "field",
            **"displayType" : "TEXTAREA",**
            "description" : "Contract EU Funds Programme",
            "_label" : "field|name|BT-722-Contract"

in fields.json : 
 "id" : "BT-722-Contract",
    "parentNodeId" : "ND-ContractEUFunds",
    "name" : "Contract EU Funds Programme",
    "btId" : "BT-722",
    "xpathAbsolute" : "/*/ext:UBLExtensions/ext:UBLExtension/ext:ExtensionContent/efext:EformsExtension/efac:NoticeResult/efac:SettledContract/efac:Funding/cbc:FundingProgramCode",
    "xpathRelative" : "cbc:FundingProgramCode",
    **"type" : "code",
    "legalType" : "CODE",**
    "repeatable" : {
      "value" : false,
      "severity" : "ERROR"
    },

bye
Renato

**Comment by YvesJo** [comment]
I think this is a similar issue as https://github.com/OP-TED/eForms-SDK/discussions/411 which should get fixed with SDK 1.8, may be @rouschr may confirm.

**Reply by YvesJo** [reply, marked as answer]
Confirmed that this is the same as #411
